### PR TITLE
chore(deps): update cypress packages and add network shim captures

### PR DESCRIPTION
**Pull request by mediremi**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -15,20 +15,24 @@
...
+        "cypress:open": "d2-utils-cypress open --appStart 'yarn cypress:start'",
+        "cypress:run": "d2-utils-cypress run --appStart 'yarn cypress:start'",
+        "cypress:capture": "d2-utils-cypress run --capture --appStart  'yarn cypress:start' --serverMinorVersion 37",
+        "cypress:stub": "d2-utils-cypress run --stub --appStart  'yarn cypress:start' --serverMinorVersion 37"
     },
     "devDependencies": {
         "@dhis2/cli-app-scripts": "^6.1.3",
         "@dhis2/cli-style": "^7.3.0",
-        "@dhis2/cli-utils-cypress": "^2.2.2",
+        "@dhis2/cli-utils-cypress": "^8.0.0-alpha.3",
```
Comment (varl) [inline package.json:27]: This can be bumped to the latest version (not alpha).

File: package.json [hunk 2]
```diff
@@ -15,20 +15,24 @@
...
         "start": "d2-app-scripts start",
         "test": "d2-app-scripts test",
-        "cy:start": "BROWSER=none yarn start",
-        "cy:open": "d2-utils-cypress open --appStart 'yarn cy:start'",
-        "cy:run": "d2-utils-cypress run --appStart 'yarn cy:start'"
+        "cypress:start": "BROWSER=none REACT_APP_DHIS2_BASE_URL=https://debug.dhis2.org/dev REACT_APP_DHIS2_API_VERSION=37 yarn start",
+        "cypress:open": "d2-utils-cypress open --appStart 'yarn cypress:start'",
+        "cypress:run": "d2-utils-cypress run --appStart 'yarn cypress:start'",
+        "cypress:capture": "d2-utils-cypress run --capture --appStart  'yarn cypress:start' --serverMinorVersion 37",
+        "cypress:stub": "d2-utils-cypress run --stub --appStart  'yarn cypress:start' --serverMinorVersion 37"
```
Comment (varl) [inline package.json:22]: These need to be adjusted, `--appStart`  has been removed.

@ismay used a helper tool from one of the Cypress devs in scheduler: https://github.com/dhis2/scheduler-app/blob/master/package.json#L14-L16

You may also need to adjust the e2e job in the workflow: ref https://github.com/dhis2/scheduler-app/blob/master/.github/workflows/dhis2-verify-app.yml#L66-L92

File: package.json [hunk 3]
```diff
@@ -15,20 +15,24 @@
...
-        "@dhis2/cli-utils-cypress": "^2.2.2",
+        "@dhis2/cli-utils-cypress": "^8.0.0-alpha.3",
         "@testing-library/react-hooks": "^3.4.1",
         "fake-indexeddb": "^3.0.2",
         "react-test-renderer": "^16.13.1"
     },
     "dependencies": {
         "@dhis2/app-runtime": "^2.8.0",
+        "@dhis2/cypress-commands": "^8.0.0-alpha.3",
+        "@dhis2/cypress-plugins": "^8.0.0-alpha.3",
```
Comment (varl) [inline package.json:35]: Also bump.

**Review by varl — COMMENTED** [review]
A few suggestions added inline.

**Comment by mediremi** [comment]
@varl I've bumped the cypress-related dependencies and have updated the workflow. [The only thing missing now is a cypress `projectId`](https://github.com/dhis2/cache-cleaner-app/pull/478/checks?check_run_id=2961777826#step:4:121)

**Comment by varl** [comment]
> @varl I've bumped the cypress-related dependencies and have updated the workflow. [The only thing missing now is a cypress `projectId`](https://github.com/dhis2/cache-cleaner-app/pull/478/checks?check_run_id=2961777826#step:4:121)

You can add this to the `cypress.json`:

```json
{
    "projectId": "z3t5e2"
}
```

**Comment by varl** [comment]
I added the secret `CYPRESS_RECORD_KEY` to this repo.

**Comment by mediremi** [comment]
The issue now seems to be that `dhis2_datatest_prefix` needs to be set to `dhis2-cachecleaner`. Should I remove that prefix from the various `dataTest` props or does this need to be added as an env to the workflow?

**Comment by Mohammer5** [comment]
You can set it in the `cypress.json`, you can check out the sms config app's [`cypress.json`](https://github.com/dhis2/sms-configuration-app/blob/master/cypress.json)

**Review by Mohammer5 — APPROVED** [review]
LGTM

**Pull request merged**